### ISSUE TITLE
NTR: Antistreptolysin O titer high (Lab: serology & immunological tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8712]
New term request Antistreptolysin O titer high (hasson_100108160033): ASO; ASLO; group A streptococcus infection history  UMLS: 0860719

**Comment by pnrobinson** [comment]
@MickeySegal
Given the multitude of ways of reporting tests like this (high, titer=1/128, etc etc), I think it is best to have terms be called
Positive Antistreptolysin O antibody positivity and then have modifers to describe IgG vs IgM and strength of the finding. Otherwise there would be a true explosion of different terms...
Let me know if this would be an issue on your side or if you have other suggestions?

**Comment by MickeySegal** [comment]
That is fine, but having both positive and positivity in the name would seem odd.

**Comment by pnrobinson** [comment]
Your right, my mistake. Let's do **Antistreptolysin O antibody positivity**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000348
- label: Antistreptolysin O antibody positivity
- parents: [Positive bloodstream antibacterial antibody positivity (HP:0430061)](https://hpo.jax.org/app/browse/term/HP:0430061)
- pmid: 
- def: The presence of antibodies in the blood circulation that react against streptolysin O.
- comment: Antistreptolysin O (ASO) is an antibody against streptolysin O that is generally produced by Group A Streptococcus (GAS). GAS causes a diverse range of human infections, like pharyngitis, cellulitis, scarlet fever, and others. Moreover, a GAS infection can trigger postinfectious immune-mediated disorders, including acute poststreptococcal glomerulonephritis, acute rheumatic fever, and rheumatic heart disease.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8712
- synonyms:
